app/services/preserved_object_handler.rb

```diff
@@ -110,14 +104,11 @@ def update_version
     if invalid?
       results << result_hash(INVALID_ARGUMENTS, errors.full_messages)
     else
-      Rails.logger.debug "update_version #{druid} called and druid in Catalog"
-      upd_results = with_active_record_rescue do
-        if endpoint.endpoint_type.endpoint_class == 'online'
-          results.concat update_online_version
-        elsif endpoint.endpoint_type.endpoint_class == 'archive'
-        end
+      Rails.logger.debug "update_version #{druid} called"
```

Comment:
why remove the specificity of the debug logging messages when they are still in the same different conditions?

Reply:
at the time of the message, it's not clear that the object actually exists, since we haven't checked for that.  though i believe we were checking for existence when that log message was initially added.